Sources/Subprocess/Subprocess.swift

```diff
@@ -26,147 +26,628 @@
 //
 
 import Foundation
+import Combine
```

Comment:
is Combine used ?

Reply:
Yes, `TopLevelDecoder`. Which was always intended to be in stdlib… https://forums.swift.org/t/move-combines-toplevelencoder-and-topleveldecoder-protocols-into-the-standard-library/32494/46